stdlib/public/Distributed/DistributedActor.swift

```diff
@@ -35,7 +35,7 @@ public protocol AnyActor: AnyObject {}
 /// which involves enqueuing new partial tasks to be executed at some
 /// point.
 @available(SwiftStdlib 5.5, *)
-public protocol DistributedActor: AnyActor, Codable {
+public protocol DistributedActor: AnyActor, Codable, Hashable, Identifiable {
```

Comment:
For reference, identifiable is:

```swift
public protocol Identifiable {
  associatedtype ID: Hashable

  /// The stable identity of the entity associated with this instance.
  var id: ID { get }
}
```

so the ID MUST be Hashable, which forces us into the `id` cannot be a protocol, because `Hashable` has a `Self` requirement... This forces us into implementing it as:

```
nonisolated var id: AnyActorIdentity
```

where `AnyActorIdentity` is the type eraser `struct`. We must inside it implement the equals and hashcode by delegating to the protocol. The `ActorIdentity` _cannot_ be Hashable because we cannot store it that way.